phpstan.neon

```diff
@@ -1,13 +1,9 @@
 parameters:
-    tmpDir: var/cache/ecs
     level: 5
 
     paths:
         - src
 
-    symfony:
-        container_xml_path: '%rootDir%/../../../var/cache/dev/Bolt_KernelDevDebugContainer.xml'
-
```

Comment:
This actually disables the idea of **static** analysis. Now the whole database connection + configuration and fully built dependency container must be working, before PHPStan can even find a missing method. 

It's only possible inside container, which is kind-off overkill for static analysis.

I personally do not use this lock, as it slowls down the feedback and couples layers that should be separated.